docs/conf.py

```diff
@@ -315,3 +317,9 @@ def word_filter(self, word):
     # This is an example GitHub URL
     r'https://github.com/ClusterHQ/flocker/compare/release/flocker-1.2.3...release-maintenance/flocker-1.2.3/fix-a-bug-FLOC-1234\?expand=1'
 ]
+
+# Define some custom global variables
+# See http://sphinx.readthedocs.org/en/latest/config.html#confval-rst_epilog
+rst_epilog = """
+.. |digitalocean_kernel_title| replace:: ``{}``
```

Comment:
:question: Note that if you prefer I don't think you need the underscores and can instead have spaces in identifiers. Examples at http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#substitution-definitions.